osgar/drivers/system_monitor.py

```diff
@@ -3,19 +3,47 @@
 """
 
 import sys
+import subprocess
 import psutil
 # https://psutil.readthedocs.io/en/latest/#system-related-functions
 from threading import Thread
 
+
+def get_timestamp(msg):
+    return float(msg.split(b"]")[0][1:])
+
 class SystemMonitor:
     def __init__(self, config, bus):
         self.input_thread = Thread(target=self.run_input, daemon=True)
         self.bus = bus
         bus.register('cpu', 'ram', 'temp')
         self.sleep = config.get('sleep', 1)
+        self.log_dmesg = config.get('dmesg', False)
+        self.last_dmesg_time = None
+        if self.log_dmesg:
+            bus.register('dmesg')
```

Comment:
I was not even sure that you can call `register()` more than once?? I would add it to outputs always